website/project/model.py

```diff
@@ -204,7 +230,9 @@ class Comment(GuidStoredObject, SpamMixin, Commentable):
     # The type of root_target: node/files
     page = fields.StringField()
     content = fields.StringField(required=True,
-                                 validate=[MaxLengthValidator(settings.COMMENT_MAXLENGTH), validators.string_required])
+                                 validate=[validators.comment_maxlength(settings.COMMENT_MAXLENGTH), validators.string_required])
+    # The mentioned users
+    ever_mentioned = fields.ListField(fields.StringField())
```

Comment:
In other words, `ever_mentioned` was originally meant to contrast with `new_mentions`. Since there's now only one attribute instead of two, it could make sense to simplify to `mentions`.